plugins/aap-backend/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@janus-idp/backstage-plugin-aap-backend",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "backend-plugin"
+  },
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test --passWithNoTests --coverage",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "dependencies": {
+    "@backstage/backend-common": "^0.18.5",
+    "@backstage/backend-tasks": "^0.5.2",
+    "@backstage/catalog-model": "^1.3.0",
+    "@backstage/config": "^1.0.7",
+    "@backstage/plugin-catalog-node": "^1.3.7",
+    "winston": "^3.2.1"
+  },
+  "devDependencies": {
+    "@backstage/cli": "0.22.7",
+    "@types/supertest": "2.0.12",
+    "supertest": "6.3.3",
+    "msw": "1.2.2"
```

Comment:
Since #530 is merged now, can you please bump these as well?

https://github.com/janus-idp/backstage-plugins/blob/main/packages/backend/package.json